homeassistant/components/prusalink/sensor.py

```diff
@@ -146,13 +146,15 @@ class PrusaLinkSensorEntityDescription(
             translation_key="progress",
             native_unit_of_measurement=PERCENTAGE,
             value_fn=lambda data: cast(float, data["progress"]),
-            available_fn=lambda data: data.get("progress") is not None,
+            available_fn=lambda data: data.get("progress") is not None
```

Comment:
Please put parentheses around the lambda and indent it to make it easier to distinguish it from the entity description parameters.